docs/Public-api.md

```diff
@@ -0,0 +1,132 @@
+## pc (gpucsl/pc/pc.py)
+
+pc(
+    data: np.ndarray,
+    data_distribution: DataDistribution,
+    max_level: int,
+    alpha=0.05,
+    kernels=None,
+    is_debug: bool = False,
+    should_log: bool = False,
+    devices: List[int] = [0],
+    sync_device: int = None,
+    gaussian_correlation_matrix: np.ndarray = None,
+    discrete_max_memory_size=None,
+) -> PCResult
+
+Executes the PC algorithm.
+
+### Parameters
+- data: The data you want to analyze  
+- data_distribution: Either DataDistribution.DISCRETE or DataDistribution.GAUSSIAN depending on the assumed distribution of your data  
+- max_level: max level until which the pc algorithm will run. Depending on the max level data will get allocated on the GPU, so you want to keep it small to avoid out of memory problems  
```

Comment:
```suggestion
- max_level: max level until which the pc algorithm will run (inclusive). Depending on the max level data structures will get allocated on the GPU, so you want to keep it small to avoid out of memory problems  
```